### DISCUSSION TITLE
Persist app volumes or provide global startup default

### DISCUSSION THREAD

**Discussion by wobblyfootgamer** [File-New-Project/EarTrumpet#652] [Ideas]
Hi Guys,

First of all, I want to say that eartrumpet is fantastic. I use it all the time and without the control for the audio levels it provides then it would be a little more difficult to make videos.

Can I make a suggestion for a new function please?

Could it be possible to add a function to set the default volume for all apps?

For example, I would like to have the default volume level set to 70 so that when I record videos, I know that my mic volume is slightly higher thus making my commentary more audible. At the moment everything is set to 100 with the windows volume control slider set to 70 and it would be absolutely great to be able to set a default volume level.

Prior to using eartrumpet, I had to set my playback volumes to level 15 but eartrumpet makes it so much more manageable. I would even pay for the app.

Regards
Craig

**Comment by riverar** [comment]
We'll take a look, but I don't think Windows itself supports this. Makes sense though!

**Comment by wobblyfootgamer** [comment]
Hi,

That is awesome.

Thank You
Craig Grant
aka wobblyfootgamer
